tests/queries/0_stateless/03401_get_merge_tree_setting.sql

```diff
@@ -0,0 +1,19 @@
+SELECT
+    toBool(t1.val = t2.val) AS should_be_equal
+FROM
+    (SELECT toString(value) AS val FROM system.merge_tree_settings WHERE name = 'index_granularity') AS t1,
+    (SELECT toString(getMergeTreeSetting('index_granularity')) AS val) AS t2;
+
+SELECT
+    toBool(t1.val = t2.val) AS should_be_equal
+FROM
+    (SELECT toString(value) AS val FROM system.merge_tree_settings WHERE name = 'max_merge_selecting_sleep_ms') AS t1,
+    (SELECT toString(getMergeTreeSetting('max_merge_selecting_sleep_ms')) AS val) AS t2;
+
+SELECT ('TEST INVALID ARGUMENTS');
+
+SELECT getMergeTreeSetting(10); -- { serverError ILLEGAL_TYPE_OF_ARGUMENT }
```

Comment:
Can we also test two more cases 1. with more than ` arguments and 2. no arguments just to ensure that the validation in code works as expected? Applies for both the functions.

Reply:
Test cases had been added